packages/react-cards/src/next/useCardClasses.ts

```diff
@@ -1,178 +1,270 @@
+          borderColor: 'var(--card-hovered-borderColor, var(--card-borderColor))',
+          borderWidth: 'var(--card-hovered-borderWidth, var(--card-borderWidth))',
+          boxShadow: 'var(--card-hovered-boxShadow, var(--card-boxShadow))',
+          color: 'var(--card-hovered-contentColor, var(--card-contentColor))',
+        },
+
+        '&:active': {
+          background: 'var(--card-pressed-background, var(--card-background))',
+          borderColor: 'var(--card-pressed-borderColor, var(--card-borderColor))',
+          borderWidth: 'var(--card-pressed-borderWidth, var(--card-borderWidth))',
+          boxShadow: 'var(--card-pressed-boxShadow, var(--card-boxShadow))',
+          color: 'var(--card-pressed-contentColor, var(--card-contentColor))',
+        },
+
+        '&[aria-disabled=true]': {
+          background: 'var(--card-disabled-background)',
+          borderColor: 'var(--card-disabled-borderColor)',
+          borderWidth: 'var(--card-disabled-borderWidth)',
+          boxShadow: 'var(--card-disabled-boxShadow)',
+          color: 'var(--card-disabled-contentColor)',
+          cursor: 'var(--card-disabled-cursor)',
+        },
+      },
+    ],
 
-    '&:active': {
-      '--card-pressed-backgroundColor': 'var(--card-disabled-backgroundColor)',
-      '--card-pressed-borderColor': 'var(--card-disabled-borderColor)',
-      '--card-pressed-borderWidth': 'var(--card-disabled-borderWidth)',
-      '--card-pressed-boxShadow': 'var(--card-disabled-boxShadow)',
+    _selected: {
+      background: 'var(--card-selected-background)',
+      borderColor: 'var(--card-selected-borderColor)',
+      borderWidth: 'var(--card-selected-borderWidth)',
+      boxShadow: 'var(--card-selected-boxShadow)',
+      color: 'var(--card-selected-contentColor)',
     },
   },
 
-  _fluid: {
-    '--card-height': 'var(--card-fluid-height)',
-    '--card-width': 'var(--card-fluid-width)',
-  },
+  variants: (theme: Theme): CardVariants => {
+    return {
+      root: {
+        size: {
+          smallest: {
+            borderRadius: '4px',
+            height: 'auto',
+            margin: '0',
+            padding: '8px',
+            width: 'auto',
+          },
+          smaller: {
+            borderRadius: '4px',
+            height: 'auto',
+            margin: '0',
+            padding: '8px',
+            width: 'auto',
+          },
+          small: {
+            borderRadius: '4px',
+            height: 'auto',
+            margin: '0',
+            padding: '8px',
+            width: 'auto',
+          },
+          medium: {
+            borderRadius: '4px',
+            height: 'auto',
+            margin: '0',
+            padding: '16px',
+            width: 'auto',
+          },
+          large: {
+            borderRadius: '6px',
+            height: 'auto',
+            margin: '0',
+            padding: '16px',
+            width: 'auto',
+          },
```

Comment:
seems like you have 2 configurations here, would shave off some bundle size to just create a const for the 2 configurations and reference them.

Reply:
Done!